src/controllers/__tests__/authenticationController.test.ts

```diff
@@ -528,6 +532,63 @@ describe("AuthenticationController#acs", () => {
       expectedNewProfile
     );
   });
+
+  it("should return unauthorized using a CIE test environment with no whitelisted user fiscalcode", async () => {
+    const res = mockRes();
+    const anInvalidCieTestUser = {
+      ...validUserPayload,
+      fiscalNumber: anotherFiscalCode,
+      issuer: Object.keys(CIE_IDP_IDENTIFIERS)[0]
+    };
+
+    const response = await controller.acs(anInvalidCieTestUser);
+    response.apply(res);
+
+    expect(controller).toBeTruthy();
+    expect(res.status).toHaveBeenCalledWith(403);
+  });
+
+  it.each`
+    title                                   | fiscalNumber                     | issuer
+    ${"if a CIE TEST user is in whitelist"} | ${validUserPayload.fiscalNumber} | ${Object.keys(CIE_IDP_IDENTIFIERS)[0]}
```

Comment:
Looking at https://github.com/pagopa/io-spid-commons/blob/fbfed7fe64e186dc1f50822fd2e95e272f6b4a56/src/config.ts#L15 I see that the test issuer is the [2] and prod issuer is the [1]. 

I think you can use just those indexes without hardcoding.